codalab/worker/main.py

```diff
@@ -133,13 +133,9 @@ def parse_args():
     )
     parser.add_argument(
         '--bundle-runtime',
-        choices=[
-            BundleRuntime.DOCKER.value,
-            BundleRuntime.KUBERNETES.value,
-            BundleRuntime.SINGULARITY.value,
-        ],
+        choices=[BundleRuntime.DOCKER.value, BundleRuntime.KUBERNETES.value,],
```

Comment:
Why remove singularity here? I see the referenced issue, but it doesn't detail why this should be removed.

Reply:
Singularity support was something we had worked on but was unfinished, and we don't need it anymore, so we'd like to remove it.